package.json

```diff
@@ -36,7 +36,9 @@
     "lint": "eslint --ext .ts . && prettier -c src test demo",
     "prepack": "pnpm build",
     "release": "pnpm test && changelogen --release && git push --follow-tags && pnpm publish",
-    "test": "pnpm lint && vitest run --coverage",
+    "test": "pnpm lint && run-p -r azurite-table-storage vitest",
+    "vitest": "vitest run --coverage",
+    "azurite-table-storage": "azurite --silent",
```

Comment:
seems unused. (was it for local testing?)

Reply:
I'm not sure what you mean, they are all 3 used in the `test` task :) Azurite is the Azure storage emulator and is started before vitest runs to emulate table and other storage endpoints. If you think it is overhead for the test pipeline, I can remove it and try to do some mocking for that.

Reply:
Hmm right. What if we run it in a beforeAll step using execa and then kill the process after?

Reply:
I fiddled around a lot with the npm package and tried to run it with node `spawn` at first, but it's not really made to be run like this and had a lot of inconsistent issues and problems being started and killed properly. I can try to mock the replies, as I would like to avoid spawning it from node.

Reply:
Got it to work with exec